tests/cross/src/test/scala/tests/pc/CompletionMatchSuite.scala

```diff
@@ -43,7 +43,8 @@ object CompletionMatchSuite extends BaseCompletionSuite {
       |case object Weekend extends Weekday
       |@@
       |""".stripMargin,
-    ""
+    "",
+    filter = _ => false
```

Comment:
Is this used any other way than `_ => false` or `_=> true` ? It could just be a flag then

Reply:
The `filter: String => Boolean` parameter is identical to the `checkEditLine` method.